### ISSUE TITLE
Link text should be underlined or bold for accessibility

### ISSUE THREAD

**Issue by ShrimpCryptid** [allen-cell-animated/timelapse-colorizer#400]
## Description

![image](https://github.com/allen-cell-animated/timelapse-colorizer/assets/30200665/95080edf-1f78-44af-8cf2-81ea5e05fd22)

Links are currently blue but have no additional styling, which does not align with WCAG rules. They should either be underlined or bold to increase contrast. Currently the built-in browser focus styling (box selection) also does not appear, and underlining is used as the focus style instead.

TODO discuss with @lynwilhelm 

## Expected Behavior

_What did you expect to happen instead?_

## Reproduction

_A minimal example that exhibits the behavior._

## Environment

_Any additional information about your environment. E.g. OS version, python version, browser etc._

**Comment by ShrimpCryptid** [comment]
Closed by #457!
